test/utils.js

```diff
@@ -32,7 +32,7 @@ const ModuleHelper = require('../src/builder/ModuleHelper.js');
 function initGit(dir, remote, branch) {
   const pwd = shell.pwd();
   shell.cd(dir);
-  shell.exec('git init');
+  shell.exec('git init -b master');
```

Comment:
I have `main` configured as default branch in my `.gitconfig`, so the tests fail locally:

```ini
[init]
defaultBranch = main
```